src/support/libsupportinit.c

```diff
@@ -12,9 +12,7 @@ static int isInitialized = 0;
 void libsupport_init(void)
 {
     if (!isInitialized) {
-#ifdef _OS_WINDOWS_
-        SetConsoleCP(1252); // ANSI Latin1; Western European (Windows)
-#endif
+
```

Comment:
this feels slightly risky and unrelated? (reverts eac525c89f463b2721f348a5870a81f7b431d6be, but libuv is now unicode-aware, so perhaps that's not as bad now)

Reply:
We need to remove this or set the console to `SetConsoleCP(CP_UTF8)` or else the sequences don't get interpreted correctly

Reply:
CP_UTF8 (65001) is not well supported. That's why we set it to 1252 (It even can break `dir`: https://github.com/nodejs/node-v0.x-archive/issues/4246#issuecomment-10142715)

Reply:
Is that comment  relevant? It looks like part of the issue was with using a raster front

Reply:
Looking at  https://github.com/JuliaLang/julia/issues/6795

```
print("测试")
测试

julia> print("\346\265\213\350\257\225")
测试
```

With this PR

Reply:
Looks like things probably got fixed internally in libuv?

Reply:
So I think this is safe.

Reply:
Yes, libuv Is fixed now to bypass this. The issue would be with other shared libraries that assume an ascii-compatible stream.

Reply:
Good to hear. Is that issue more theoretical than practically relevant (?), I don't know what we could do about this pathological case without regressing the whole system to be dumber because of some library is forcing ascii.